packages/fast-form-generator-react/src/utilities/form-input.style.ts

```diff
@@ -466,7 +467,9 @@ export function applyAddItemStyle(): ICSSRules<{}> {
             background: pinkPlus,
             width: toPx(16),
             height: toPx(16),
-            left: "0"
+            left: "0",
+            top: "50%",
+            transform: "translateY(-50%)"
```

Comment:
I assume this guarantees that the glyph is positioned center vertically in its parent?

Reply:
Correct.